setup.py

```diff
@@ -84,7 +84,7 @@
     },
     install_requires=[
         "flask-caching<1.11",  # https://github.com/equinor/webviz-subsurface/issues/1043
-        "dash>=2.0.0",
+        "dash>=2.6",
```

Comment:
Is it necessary to change this? the dash._callback NoUpdate existed in the 2.0.0 tag as well as far as I can see. 
https://github.com/plotly/dash/blob/2975e001cf017919929b0ebad1f1d1e14fa32f93/dash/_callback.py#L25

Reply:
Good catch. Indeed it does (for some strange reason) - I'll keep the current lower limit then. Will fix.